wagn/lib/wagn/mods_spec_helper.rb

```diff
@@ -1,6 +1,5 @@
 # -*- encoding : utf-8 -*-
 require 'spork'
-raise "is this loaded anywhere?"
```

Comment:
Yes, I realize that.  The question I'm asking is whether the spec template could refer to the probably almost identical file in the card gem.  There may be a good reason for the split, but more likely it just happened with the gem split.